optimade/server/models.py

```diff
@@ -53,24 +58,110 @@ class StructureResource(EntryResource):
     attributes: StructureResourceAttributes
 
 
-class OptimadeResponseMeta(BaseModel):
+class OptimadeProvider(BaseModel):
+    """ Stores information on the database provider of the
+    implementation.
+
     """
-    A JSON API meta member that contains JSON API meta objects of non-standard meta-information.
 
-    In addition to the required fields, it MAY contain
+    name: str = Schema(
+        ...,
+        description="a short name for the database provider"
+    )
+
+    description: str = Schema(
+        ...,
+        description="a longer description of the database provider"
+    )
 
-    - `data_available`: an integer containing the total number of data objects available in the database.
-    - `last_id`: a string containing the last ID returned.
-    - `response_message`: response string from the server.
+    prefix: str = Schema(
+        ...,
+        description="database-provider-specific prefix as found in "
+                    "Appendix 1."
```

Comment:
since we now have a class hierarchy and could be reusing the "prefix" schema, would it make sense to remove references to the appendix and just put the info in here?

Reply:
I agree, but I think this is worth another PR as it will require some refactoring of other files